### ISSUE TITLE
您好 运行您的代码出现prior_Estimation_Network输出的通道和heatmaps通道数不匹配

### ISSUE THREAD

**Issue by zkj1995** [cydiachen/FSRNET_pytorch#10]
前者为128 后者为11，通道数不匹配无法求MSE-loss，我目前打算最后的hourglass block后加一个1x1 的卷积进行降维到通道11。但是又担心引起后面一连串的通道数不匹配的问题。
请问 您的prior_Estimation_Network输出的通道和heatmaps通道数是否都是11？
这个问题怎么解决？

**Comment by zkj1995** [comment]
问题已经解决，正在训练

**Comment by zhangluustb** [comment]
请问你直接加了一个1*1的卷积的吗？

**Comment by yuanlunxi** [comment]
> 问题已经解决，正在训练

@zkj1995 你好，你是通过降维来解决这个问题的么？

**Comment by zkj1995** [comment]
@zhangluustb 是的

**Comment by zkj1995** [comment]
@yuanlunxi 对啊降维到11

**Comment by yuanlunxi** [comment]
@zkj1995 那这个特征不是还要concat到fine encode那个网络么，会不会影响效果呢？你现在复现论文 的效果了吗？

**Comment by zkj1995** [comment]
@yuanlunxi 先验网络最后是concat在fine encode，这里为了通道数匹配所以降维。这个代码，我跑了，效果还可以，但是对于老人的人脸生成效果不好，太过平滑，脸部褶皱不明显。

**Comment by yuanlunxi** [comment]
@zkj1995
那你concat的prior feature是不是也变成了11维了？另外这份代码我看实现的是fsrnet，你有跑fsrgan么？
按照你的方法我也跑起来了，可以看一下你的loss曲线么？我好有个参照。

**Comment by Gnaixzl** [comment]
> @yuanlunxi先验网络最后是concat在fine encoding，这里为了通道数匹配所以降维。这个代码，我跑了，效果还可以，但是对于老人的人脸生成效果不好，太过光滑，脸部皱纹不明显。

你好，代码能否分享一下？您有实现fsrgan的复现吗？谢谢

**Comment by vthmrd** [comment]
@zkj1995 @yuanlunxi 您好，请问您实现的代码可以分享一下吗？谢谢您

**Comment by ghost** [comment]
> @yuanlunxi 先验网络最后是concat在fine encode，这里为了通道数匹配所以降维。这个代码，我跑了，效果还可以，但是对于老人的人脸生成效果不好，太过平滑，脸部褶皱不明显。

您好，能问下这个怎么实现吗？

**Comment by ghost** [comment]
> 前者为128 后者为11，通道数不匹配无法求MSE-loss，我目前打算最后的hourglass block后加一个1x1 的卷积进行降维到通道11。但是又担心引起后面一连串的通道数不匹配的问题。
> 请问 您的prior_Estimation_Network输出的通道和heatmaps通道数是否都是11？
> 这个问题怎么解决？

您好，能否提供一下代码？

**Comment by Tony-Tseng** [comment]
![Screen Shot 2020-05-18 at 4 27 49 PM](https://user-images.githubusercontent.com/35694556/82191075-940d6680-9924-11ea-8c24-9f2d0e3f1813.png)
I can train the model after modifying networks.py.

**Comment by SJshow** [comment]
> ![屏幕快照2020-05-18在4 27 49 PM](https://user-images.githubusercontent.com/35694556/82191075-940d6680-9924-11ea-8c24-9f2d0e3f1813.png)
> 修改network.py之后，我可以训练模型。

after modifying the prior network, torch.cat ((parsing, heatmaps), 1)   there is a channel mismatch problem, how do you solve it?

**Comment by Tony-Tseng** [comment]
<img width="1093" alt="Screen Shot 2020-06-14 at 10 03 32 PM" src="https://user-images.githubusercontent.com/35694556/84595569-1d17af00-ae8b-11ea-91de-1f4d49fd59ae.png">
We also need to modify the input channel of the fine SR Decoder.
